OTAnalytics/plugin_prototypes/track_visualization/track_viz.py

```diff
@@ -451,6 +487,7 @@ def _plot_dataframe(self, track_df: DataFrame, axes: Axes) -> None:
             alpha (float): transparency of the lines
             axes (Axes): axes to plot on
         """
+        print(track_df["classification"].dtype)
```

Comment:
Is this debug statement still necessary or can it be removed?

Reply:
Ups, can be removed :)